book/09-git-and-other-scms/sections/import-svn.asc

```diff
@@ -26,6 +26,8 @@ $ svn log --xml --quiet | grep author | sort -u | \
   perl -pe 's/.*>(.*?)<.*/$1 = /'
 ----
 
+Windows users can find advice and a sample command for creating an authors list here: https://docs.microsoft.com/en-us/azure/devops/repos/git/perform-migration-from-svn-to-git?view=azure-devops
```

Comment:
🤔 Could you enclose this in a `[NOTE]`, and change the text to be something like this?

```suggestion
If you're trying this on a Windows machine, this is the point where you'll run into trouble.
Microsoft have provided some good advice and samples at https://docs.microsoft.com/en-us/azure/devops/repos/git/perform-migration-from-svn-to-git[].
```